src/frontend/src/components/Post/telescope-post-content.css

```diff
@@ -19,7 +19,8 @@
   max-width: 100%;
   padding-top: 1.5rem;
   padding-bottom: 1rem;
-  width: 100%;
+  width: auto;
+  display: block;
```

Comment:
I'm not sure about this change to `block`.  This is going to mess-up cases where we have inline images in posts.  @cindyledev or @agarcia-caicedo might know more.  Am I wrong on this?